Tests/Unit/TestNGNetUtlilities.m

```diff
@@ -0,0 +1,89 @@
+/* TestNGNetUtilities.m - this file is part of SOGo
+ *
+ * Copyright (C) 2022 Nicolas Höft
```

Comment:
As you signed the CLA, please replace your name by `Inverse inc.` and remove the `Author` line.

Reply:
Yes, with the CLA, I granted inverse an extra license. But that doesnt change the authorship. I can add inverse to the copyright line. But I think keeping my author attribution is just fair